styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -129,20 +139,35 @@
   }
 
   @Override
-  public String createExecution(final RunState runState, final String execName, final FlyteExecConf flyteExecConf,
-                                final Map<String, String> annotations)
+  public String createExecution(final RunState runState, final String execName, final FlyteExecConf flyteExecConf)
       throws CreateExecutionException {
     requireNonNull(runState, "runState");
     requireNonNull(execName, "name");
     requireNonNull(flyteExecConf, "flyteExecConf");
-    requireNonNull(annotations, "annotations");
     final var launchPlanIdentifier = flyteExecConf.referenceId();
     final var execMode = runState.data().trigger()
         .map(this::toFlyteExecutionMode)
         .filter(mode -> mode != ExecutionMode.UNRECOGNIZED)
         .orElseThrow(() -> new CreateExecutionException("Missing trigger or unknown in StateData: " + runState.data()));
 
-    var extraDefaultInputs = getExtraDefaultInputs(runState.workflowInstance(), runState.data());
+    var styxVariables = getStyxVariables(runState.workflowInstance(), runState.data());
+    var triggeredParams = runState.data().triggerParameters()
+        .map(FlyteAdminClientRunner::getFilteredTriggerParams)
+        .orElseGet(Map::of);
+
+    var labels = Map.of(STYX_EXECUTION_ID, normalize(styxVariables.get(STYX_EXECUTION_ID)));
+    final var annotations = ImmutableMap.<String, String>builder()
+        .putAll(styxVariables)
+        // just to keep compatibility with removing dangling executions
+        // TODO: base dangling logic on STYX_WORKFLOW_ID, STYX_PARAMETER and STYX_EXECUTION_ID
+        //   and the remove these annotations
+        .put(STYX_WORKFLOW_INSTANCE_ANNOTATION, styxVariables.get(STYX_WORKFLOW_ID) + "#" + styxVariables.get(STYX_PARAMETER))
+        .put(STYX_EXECUTION_ID_ANNOTATION, styxVariables.get(STYX_EXECUTION_ID))
```

Comment:
Should be also normalize the `id` to be consistent with line 164?

Reply:
Same answer here, it is not needed